### PR TITLE
Code touchups

### PR DESCRIPTION
**Pull request by NightScript370**
You don't need to do one lined'd code statements in {} and RichEmbed is chainable.
Also, check if users can even send messages in that channel

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -7,61 +7,50 @@ const bot = new Discord.Client();
 const token = process.env.TOKEN;
 const prefix = process.env.PREFIX;
 
-bot.on('ready', () => {
-	console.log(`Logged in as ${bot.user.username}!`);
-});
-
+bot.on('ready', () => console.log(`Logged in as ${bot.user.username}!`));
```
Comment (MicroDroid) [inline bot.js:10]: Get this back the way it was before :rage:

File: bot.js [hunk 2]
```diff
@@ -7,61 +7,50 @@ const bot = new Discord.Client();
...
 bot.on('message', msg => {
-	console.log(`RECV: ${msg.author.tag}@${msg.channel.name} ${msg.content}`);
```
Comment (MicroDroid) [inline bot.js:15]: Can we get this or whatever equivalent back?

File: bot.js [hunk 3]
```diff
@@ -7,61 +7,50 @@ const bot = new Discord.Client();
...
 This is a bot for https://codebottle.io/
 
-\`!search <keywords>\` Search snippets with keywords
-\`!get <id>\` Get a snippet by its ID`);
-				msg.channel.send(embed);
-				break;
+\`!search <keywords>\` - Search snippets with keywords
+\`!get <id>\` - Get a snippet by its ID`);
+			msg.channel.send(embed);
+			break;
```
Comment (MicroDroid) [inline bot.js:53]: Probably fix eslint config to allow the extra indent for switch cases

**Review by MicroDroid — CHANGES_REQUESTED** [review]

File: bot.js (hunk 3 above)
Comment (NightScript370) [reply bot.js:53]: 1. I have no idea how to do ESLint stuff
2. I didn't touch the original indentation of switch cases. I simply removed a tab from the if statement

File: bot.js (hunk 2 above)
Comment (NightScript370) [reply bot.js:15]: Why log stuff though?

File: bot.js (hunk 1 above)
Comment (NightScript370) [reply bot.js:10]: Why? It's just one line.

File: bot.js (hunk 3 above)
Comment (MicroDroid) [reply bot.js:53]: oh ok

File: bot.js (hunk 1 above)
Comment (burdoto) [reply bot.js:10]: it's prettier, though

File: bot.js (hunk 2 above)
Comment (burdoto) [reply bot.js:15]: it helps debugging :)

**Pull request merged**